lib/request-handler.js

```diff
@@ -29,6 +30,11 @@ const requestCache = new LruCache(1000);
 // Deep error handling for vendor hooks.
 const vendorDomain = domain.create();
 vendorDomain.on('error', err => {
+  Raven.captureException(err, function (sendErr) {
```

Comment:
`caputureException` is able to send [additional data](https://docs.sentry.io/clients/node/usage/#additional-data) such as request, but I have no idea how to get request here. Since Shields.is is "no tracking" we should remove data such as cookies, IP or user fro request. Having an error + a request URL would be really valuable data.

Reply:
> Since Shields.is is "no tracking" we should remove data such as cookies, IP or user fro request

Have you had a look at sentry's 'data scrubber' feature? Certain sensitive data can be removed using server-side filters in the project settings: https://docs.sentry.io/learn/sensitive-data/

:+1: +1 for using sentry btw - I've found it very helpful on other projects

Reply:
It would be more valuable to send messages to sentry from lib/log.js error as well, if we can avoid double sends.

Reply:
done